.github/workflows/devtools.yml

```diff
@@ -125,11 +125,7 @@ jobs:
       # e.g. if set 1 fails, continue with set 2 anyway
       fail-fast: false
     runs-on: macos-latest
-    env:
-      # The total number of shards. Set dynamically when length of single matrix variable is
```

Comment:
driveby: in this case it can be dynamic, because the matrix size is just the array of shards